src/components/NcSettingsSection/NcSettingsSection.vue

```diff
@@ -33,15 +33,16 @@ This component is to be used in the settings section of nextcloud.
 	<NcSettingsSection
 		name="Two-Factor Authentication"
 		description="Two-factor authentication can be enforced for all users and specific groups."
-		doc-url="https://docs.nextcloud.com/server/19/go.php?to=admin-2fa">
+		doc-url="https://docs.nextcloud.com/server/19/go.php?to=admin-2fa"
+		:limit-width="true">
```

Comment:
Why do we need it with the default value?

Reply:
I think in the past (v7 or so) it was false by default.

But I have not changed it, just revert

Reply:
Maybe remove it from the example to not show the deprecated prop?